Controllers/CmsController.cs

```diff
@@ -68,7 +69,107 @@ public async Task<IActionResult> Page(Guid id, bool draft = false)
                 return Unauthorized();
             }
         }
+        
+        /// <summary>
+        /// Gets the home page with the given id.
+        /// </summary>
+        /// <param name="id">The unique page id</param>
+        /// <param name="draft">If a draft is requested</param>
+        [Route("/homepage")]

```

Comment:
We want those to be dynamically created by the CMS, not to comply with a certain route. Please test you can add as much dynamic pages to the menu. The default homepage template route should be "/"

Reply:
Many page of the same type can be created. The route is updated. There is a middleware that translates the page`s "slug" to the route and id, so this is not the route that is displayed.